src/lib/serverOtpStorage.ts

```
@@ -0,0 +1,67 @@
+  email: string;
+  createdAt: Date;
+  expiresAt: Date;
+  attempts: number;
+  isUsed: boolean;
+}
+
+declare global {
+  var otpStorage: Map<string, StoredOTPData> | undefined;
+}
+
+const otpStorage = globalThis.otpStorage ?? new Map<string, StoredOTPData>();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'otpStorage'. Consider to delete it or rename it.

'otpStorage' is defined here:


(lint/suspicious/noRedeclare)